api/v1/runtime/runtime_agent_api.proto

```diff
@@ -359,3 +360,32 @@ message KubeLinterCheck {
   uint64 passed = 2; // Represented as bitmasks of passed checks.
   uint64 failed = 3; // Represented as bitmasks of failed checks.
 }
+
+message Process {
```

Comment:
Go structs memory layout would be more efficient if you group uint32, uint64 first.

```
message Process {
  uint32 pid = 1;
  uint32 ppid = 2;
  uint64 start_time = 3;
  uint64 parent_start_time = 4;
  uint64 exit_time = 5;
  string filepath = 6;
  repeated string args = 7;
}
```